src/Build/Evaluation/Expander.cs

```diff
@@ -3506,7 +3506,7 @@ private static void ConstructFunction(IElementLocation elementLocation, string e
                             functionArguments = ExtractFunctionArguments(elementLocation, expressionFunction, argumentsContent);
                         }
 
-                        remainder = expressionFunction.Substring(argumentsEndIndex + 1);
+                        remainder = expressionFunction.Substring(argumentsEndIndex + 1).Trim();
```

Comment:
Andy and I confirmed that `.Trim()` will not allocate if there is no trailing or ending whitespace.  So users won't pay a penalty unless they start adding newlines.